src/components/shared/AutoComplete/renderInput.tsx

```diff
@@ -20,12 +20,13 @@ const RenderInput =
     const [animatedIsFocused] = useState(new Animated.Value(on ? 1 : 0));
 
     useEffect(() => {
-      Animated.timing(animatedIsFocused, {
-        toValue: on ? 1 : 0,
-        delay: 300,
-        duration: 200,
-      }).start();
-    }, [on]);
+      const config = {
+        toValue: on ? 1 : value ? 2 : 0,
+        delay: value ? 0 : 300,
+        duration: value ? 0 : 200,
```

Comment:
@marsinearth Is there any better naming rather than `on`, `value`? I feel a bit hard to understand what they are.

Reply:
changed `on` to `focused` and `value` to use alias `label` inside `renderInput.tsx`. I need to maintain prop name `value` to get designated `text` value of `onChangeText` method of TextInput.